software/domain-watchdog.yml

```
@@ -0,0 +1,13 @@
+name: "Domain Watchdog"
+website_url: "https://github.com/maelgangloff/domain-watchdog"
+source_code_url: "https://github.com/maelgangloff/domain-watchdog"
+description: "Uses RDAP to collect publicly available info about domains, track their history, and purchase them."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - PHP
+tags:
+  - Miscellaneous
+depends_3rdparty: true
```

```suggestion
depends_3rdparty: false
```
This project does not depend on third parties to be usable, apart from RDAP servers, but that is part of the Internet infrastructure.

---

Thanks, done